STAR-Delegation/draft-ietf-acme-star-delegation.md

```diff
@@ -205,18 +205,67 @@ phase completes successfully.
 ## Delegated Identity Profile
 {: #sec-profile}
 
-This section defines a profile of the ACME protocol, to be used between the NDC and IdO.
+This section defines a profile of the ACME protocol, to be used between the NDC
+and IdO.
+
+### Delegation Configuration
+{: #sec-profile-dele-config}
+
+The configuration details regarding any delegation agreed upon by IdO and NDC
+are collected in a new delegation configuration object on the IdO side.  A
+delegation configuration object contains the CSR template (see
+{{sec-csr-template}}) that applies to that delegation.  Its structure is as
+follows:
+
+* csr-template (required, object): CSR template as defined in
+  {{sec-csr-template}}.
+
+An example delegation object is shown in {{fig-configuration-object}}.
+
+~~~
+{::include CSR-template/example-configuration-object.json}
+~~~
+{: #fig-configuration-object title="Example Delegation Configuration object"}
+
+In order to list all the delegation configuration objects that are configured
+with the NDC account, a new `delegations` attribute is added to the Account
+object.  The value of this attribute is an array of URLs each pointing to a
+delegation configuration object as shown in {{fig-account-object}}.
```

Comment:
Presumably this attribute is read-only. We should say that it can only be changed through configuration.

Reply:
Yes, this is reflected in the `Requests == none` setting in the IANA section, but will add here a note as well.